modules/salt/pages/salt-ssh.adoc

```diff
@@ -110,32 +112,40 @@ The roster content contains:
 
 == Bootstrap Sequence
 
-Salt SSH is used to bootstrap Salt clients.
+This section describes the sequence of events when clients are registered to a Salt master.
+While bootstrapping is a type of Salt SSH call, the sequence differs slightly from regular SSH calls.
+
+Bootstrapping uses Salt SSH for communication between the master and the client.
 This happens for both regular and SSH clients.
 
-The bootstrap sequence differs slightly from other Salt SSH calls.
 
-. For a regular Salt client, generate and pre-authorize the Salt key of the client
-. For an SSH client, if a proxy was selected, retrieve the SSH public key of the proxy using the `mgrutil.chain_ssh_cmd` runner.
+
+. For a regular Salt client, generate and pre-authorize the Salt key of the client.
+. For an SSH client, if a proxy was selected, retrieve the SSH public key of the proxy using the ``mgrutil.chain_ssh_cmd`` runner.
 The runner copies the public key of the proxy to the server using SSH.
-If needed it can chain multiple SSH commands to reach the proxy across multiple hops.
+If needed, it can chain multiple SSH commands to reach the proxy across multiple hops.
 . Generate pillar data for bootstrap.
-. If contact method is `ssh-push-tunnel`, fill the remote port forwarding option.
-. If the client connects through a proxy, compute the `ProxyCommand` option.
-This depends on the path used to connect to the proxy.
-For example, server to proxy1 to proxy2 to client.
-. Generate the roster for bootstrapping into a temporary file.
-. Execute this command using the Salt API:
+The pillar data is compiled and stored on the Salt master, and retrieved by the client.
```

Comment:
How is the Pillar data generated. Is the user suppose to start from a template. Are they directly creating pillar data in /srv/pillar ? Is there a way from within the Web-UI to generate the pillar data?